app/presenters/document_presenter.rb

```diff
@@ -12,7 +12,7 @@ def to_json
       base_path: document.base_path,
       title: document.title,
       description: document.summary,
-      format: document.format,
```

Comment:
If the format is "specialist_document", how will you fetch just a list of CMA Cases to show in Specialist Publisher?